src/Tickets/Commerce/Order_Modifiers/Checkout/Abstract_Fees.php

```diff
@@ -337,4 +337,17 @@ protected function append_fees_to_cart( array $items, Value $subtotal ) {
 
 		return $items;
 	}
+
+	/**
+	 * Resets the fees state and recalculates the subtotal to zero.
+	 *
+	 * This method clears the flag indicating that fees have been appended
+	 * and resets the subtotal to its default value of zero.
+	 *
+	 * @return void
+	 */
+	public function reset_fees_and_subtotal(): void {
+		self::$fees_appended = false;
+		$this->subtotal      = Value::create( 0 );
+	}
```

Comment:
I created this method because while testing the $fees_appended wasn't being reset. 

If there's another way I should handle, let me know!